Q10-Local-and-globlal-variables.md

```diff
@@ -0,0 +1,43 @@
+# What are local variables and global variables in Python?
+
+Variables declared outside of the function or in the global space are called global variables. These variables can be accessed by any function in the program.
+
+## Example
+
+We will create a global variable:
+
+```python
+e = 30
+print(e)
+
+Output: 
+30
+```
+
+Create a global variable inside the function:
+
+```python
+a = 2
+def define():
+    global b;
+    b = 4
+def add():
+    c = a + b;
+    print(c)
+define()
+add()
+```
+
+* Local Variables:
+
+Any variable declared inside a function is known as a local variable. This variable is present in the local space and not in the global space.
+
+```python
+def calculation():
+    a = 10
```

Comment:
Maybe you can name instead of `a` to `local-variable` or something similar to make it more clear about which variable you are talking about